Backoffice/Context/ContextManager.php

```diff
@@ -115,7 +115,7 @@ public function getAvailableSites()
                 foreach ($user->getGroups() as $group) {
                     /** @var SiteInterface $site */
                     $site = $group->getSite();
-                    if (null !== $site && !$site->isDeleted()) {
+                    if (null !== $site && !$group->isDeleted() && !$site->isDeleted()) {
```

Comment:
@alavieille : with this PR, what is the case where a site is deleted but not a group ?